elections57_results.py

```diff
@@ -220,6 +228,23 @@ def get_class(v):
         for c in sorted(classes):
             votes_by_class.append(votes[c])
         print '{},{}'.format(y, ','.join([str(x) for x in votes_by_class]))
+elif cmdline_args.operation == 'raw_data':
+    SALT = input("Enter salt: ") # Or any other method to get SALT known to organizers
```

Comment:
I'd take it on command line, as an option -- script runs should be reproducible,